webpack.common.ts

```diff
@@ -129,6 +129,14 @@ module.exports = {
           },
         ],
       },
+      {
+        test: /\.csv$/,
+        use: [
+          {
+            loader: 'raw-loader',
+          },
+        ],
+      },
```

Comment:
For others wondering why webpack config is being changed. With the onboarding cli, the app is serving a CSV file for users to download, and hence needs a loader.